lib/vanagon/utilities.rb

```diff
@@ -281,5 +281,16 @@ def erb_file(erbfile, outfile = nil, remove_orig = false, opts = { :binding => b
       FileUtils.rm_rf erbfile if remove_orig
       outfile
     end
+
+    # Final
+    # - 1.0.0
+    # - 3.0.5.1
+    # - 4.3.0-5
+    # Nonfinal
+    # - 1.0.0-34-g1sadjf9283
+    # - 4.2.1.2344.gfdqw83rh
+    def final_version?(version)
+      !version.match(/g[a-f0-9]{7}$/)
```

Comment:
I've been thinking about this a little bit. Would it be easier to try to parse out whether or not the version matches a specific `git describe` format -- and does this only apply to Git sources? If it does, shouldn't the logic for determining if something is final or not live there? 
I'm happy to talk my thoughts here through with you if you're interested.